root/dataframe/test_readTotemNtuple.C

```diff
@@ -0,0 +1,9 @@
+using namespace ROOT::Experimental;
+
+int test_readTotemNtuple() {
+
+  TDataFrame d("TotemNtuple", "Slimmed_TotemNTuple_9883.040.ntuple.root");
+  auto validate_x = [](double v) { std::cout << v << std::endl; };
```

Comment:
You may want to set a specific 'format' for writing the double to avoid cross-platform differences.